### ISSUE TITLE
Fullscreen mode: W menu not focusable in page/post editor with Firefox

### ISSUE THREAD

**Issue by creativecoder** [WordPress/gutenberg#29538]
## Description

When in full screen mode, the W menu is not keyboard focusable with Firefox.

This means there's no way to navigate to other parts of wp-admin (other than turning off fullscreen mode or by typing in a url).

## Step-by-step reproduction instructions

With Firefox

1. Load a page or post in the editor
2. Use tab to start navigating through the page
3. Note that the "W" menu is not focused... the first 

## Expected behaviour

The "W" menu is focused and the focus is indicated visually with an outline

## Actual behaviour

The first focused item in the header toolbar is the block inserter "+"

## WordPress information
- WordPress version: 5.6.2
- Gutenberg version: 10.1.0-rc.1
- Are all plugins except Gutenberg deactivated? Yes
- Are you using a default theme (e.g. Twenty Twenty-One)? Yes

## Device information
- Device: Desktop
- Operating system: macOS
- Browser: Firefox 86 (and Firefox Developer Edition 87.0b4)

**Comment by youknowriad** [comment]
Actually, this is not a Gutenberg bug, it seems to be a browser/OS config on MacOS. See this thread for more details. https://stackoverflow.com/questions/11704828/how-to-allow-keyboard-focus-of-links-in-firefox

Thanks for the report

**Comment by creativecoder** [comment]
Thank you! Adding the new setting to Firefox in `about:config`, by creating `accessibility.tabfocus` and setting it to `7`, fixed this for me in macOS 11.2 with Firefox 86. The system keyboard setting did not seem to have an effect in Firefox.
